src/editors/containers/VideoUploadEditor/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDropzone } from 'react-dropzone';
+import { defineMessages } from 'react-intl';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Button } from '@edx/paragon';
+import './index.scss';
+import uploadIcon from '../../data/images/upload.png';
+import infoIcon from '../../data/images/info.png';
+import * as hooks from './hooks';
+import { useDispatch } from 'react-redux';
+
+const messages = defineMessages({
+  dropVideoFileHere: {
+    defaultMessage: 'Drag and drop your videos or',
+    id: 'VideoUploadEditor.dropVideoFileHere',
+  },
+  browse: {
+    defaultMessage: 'Browse files',
+    id: 'VideoUploadEditor.browse',
+  },
+  info: {
+    id: 'VideoUploadEditor.uploadInfo',
+    defaultMessage: 'Upload .mp4 or .mov files (5 GB max)',
+  }
+});
```

Comment:
@pkulkark I think this can be extracted into `messages.js` by convention with the other editors.

Reply:
Done.